subsys/fs/Kconfig.fatfs

```diff
@@ -97,6 +97,11 @@ config FS_FATFS_CODEPAGE
 	  949 - Korean (DBCS)
 	  950 - Traditional Chinese (DBCS)
 
+config FS_FATFS_VOLUMES
+	int "Maximum number of logical drives"
+	range 1 10
+	default 8
+
```

Comment:
Please add some help to explain how does the configuration option does impact the compilation/execution.

Reply:
Added help text.